packages/search/tests/integration/components/navi-search-result/asset-test.js

```diff
@@ -43,8 +43,8 @@ module('Integration | Component | navi-search-result-asset', function(hooks) {
 
     await render(hbs`<NaviSearchResult::Asset @data={{this.result.data}}/>`);
 
-    assert.dom('li').exists({ count: 2 });
-    let results = findAll('li');
+    assert.dom('tr').exists({ count: 2 });
```

Comment:
Can we use classnames? `navi-search-result__asset`? And add an appropriate message